<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="1" failures="0" skipped="0" tests="1" time="0.103" timestamp="2023-03-21T00:12:39.209078" hostname="Jarons-MBP"><testcase classname="" name="pywhy_graphs.networkx.algorithms.causal.tests.test_m_separation" time="0.000"><error message="collection failure">ImportError while importing test module '/Users/jaron/Projects/pywhy-graphs/pywhy_graphs/networkx/algorithms/causal/tests/test_m_separation.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../networkx-dev/lib/python3.10/site-packages/_pytest/python.py:618: in _importtestmodule
    mod = import_path(self.path, mode=importmode, root=self.config.rootpath)
../../networkx-dev/lib/python3.10/site-packages/_pytest/pathlib.py:533: in import_path
    importlib.import_module(module_name)
/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
&lt;frozen importlib._bootstrap&gt;:1050: in _gcd_import
    ???
&lt;frozen importlib._bootstrap&gt;:1027: in _find_and_load
    ???
&lt;frozen importlib._bootstrap&gt;:1006: in _find_and_load_unlocked
    ???
&lt;frozen importlib._bootstrap&gt;:688: in _load_unlocked
    ???
../../networkx-dev/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:168: in exec_module
    exec(co, module.__dict__)
../../../pywhy-graphs/pywhy_graphs/networkx/algorithms/causal/tests/test_m_separation.py:3: in &lt;module&gt;
    import networkx as nx
E   ModuleNotFoundError: No module named 'networkx'</error></testcase></testsuite></testsuites>